source/_components/switcher_kis.markdown

```diff
@@ -58,25 +53,13 @@ device_password:
   description: The device's password.
   required: true
   type: string
-name:
-  description: A name of your choosing, will be used to create the entities id ('switch.switcher_kis_*name*', 'sensor.switcher_kis_*name*_schedule_3')
-  required: false
-  type: string
-  default: 'boiler'
-icon:
-  description: MDI icon, will be used for all entities.
-  required: false
-  type: icon 
 {% endconfiguration %}
 
 ## {% linkable_title Switch State Attributes %}
 
 | Attribute | Type | Description | Example |
 | --------- | ---- | ----------- | ------- |
-| `last_data_update` | datetime | The datetime of the last data updated received by the device. | 2019-02-09T13:09:12.004178 |
-| `last_state_change` | datetime | The datetime of the last change of device's state since the bridge started. | 2019-02-09T13:09:15.005698 |
-| `ip_address` | string | The IP address of the device. | '192.168.0.150' |
-| `device_name` | string | The device's configured name. | "Switcher Boiler" |
+| `friendly_name` | string | The device's configured name. | "Switcher Boiler" |
```

Comment:
Maybe say this `defaults to the device's configured name`.  If the user customizes it this value, it will be show something different.

Reply:
Thank you very much!
Fixed in this [commit](https://github.com/home-assistant/home-assistant.io/pull/8480/commits/b6696454c3e1264f42a6da7379168bcd17b273e9).
:-)